src/transformers/models/udop/processing_udop.py

```diff
@@ -16,12 +16,47 @@
 Processor class for UDOP.
 """
 
+import sys
 from typing import List, Optional, Union
 
+from transformers import logging
+
+from ...image_processing_utils import BatchFeature
 from ...image_utils import ImageInput
-from ...processing_utils import ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding, PaddingStrategy, PreTokenizedInput, TextInput, TruncationStrategy
-from ...utils import TensorType
+from ...processing_utils import ProcessingKwargs, ProcessorMixin, TextKwargs
+from ...tokenization_utils_base import PreTokenizedInput, TextInput
+
+
+if sys.version_info >= (3, 11):
+    from typing import Unpack
+else:
+    from typing_extensions import Unpack
```

Comment:
I think when https://github.com/huggingface/transformers/pull/33631 is merged we can just import here!